### PR TITLE
feat: add os string

### PR DESCRIPTION
**Pull request by romange**
Adding it both to the info log and the INFO response in the server section, similarly to how it's reported by redis.

### CODE REVIEW

File: src/server/server_family.cc [hunk 1]
```diff
@@ -428,6 +432,17 @@ void ServerFamily::Init(util::AcceptServer* acceptor, std::vector<facade::Listen
...
+
+  std::call_once(once_init, [] {
+    utsname os_name;
+
+    // Call uname() only once since it can be expensive. Cache the final result in a static string.
+    uname(&os_name);
+    os_string = StrCat(os_name.sysname, " ", os_name.release, " ", os_name.machine);
+    LOG(INFO) << "Host OS: " << os_string << " with " << shard_set->pool()->size() << " threads";
+  });
+
```
Comment (chakaz) [inline src/server/server_family.cc:445]:
```suggestion
  LOG_FIRST_N(INFO, 1) << "Host OS: " << os_string << " with " << shard_set->pool()->size() << " threads";
```

Comment (chakaz) [reply src/server/server_family.cc:445]: (or log it above within the lambda)

**Review by chakaz — APPROVED** [review]
I've proposed a suggestion below, which I think is cleaner, but that's just a proposal

File: src/server/server_family.cc [hunk 2]
```diff
@@ -371,6 +372,18 @@ void SlowLogGet(dfly::CmdArgList args, dfly::ConnectionContext* cntx, dfly::Serv
   }
 }
 
+std::string_view GetOSString() {
+  // Call uname() only once since it can be expensive. Cache the final result in a static string.
+  static string os_string = []() {
+    utsname os_name;
```
Comment (kostasrim) [inline src/server/server_family.cc:378]: that's your google trick @chakaz :eyes:

Comment (chakaz) [reply src/server/server_family.cc:378]: not mine, but yeah :)

**Review by chakaz — APPROVED** [review]

**Pull request merged**